index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    </head>
+    <body>
+        <nav><div class="logo">MY LOGO</div><div class="menu"><img src="menu.png" width="30" height="30" ></div></nav>
+        <div class="container">
+        <h1>I'm Linh Chi. Glad to see you!</h1>
+
+        <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </div>
+        <div class="connect">LET'S CONNECT</div>
+        <div class="social-media"><i class="fa-brands fa-twitter"></i><i class="fa-brands fa-linkedin-in"></i><i class="fa-brands fa-github"></i></div>
```

Comment:
:red_circle: For semantics reason, Please make sure this list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)